tests/integration/modifiers/autofocus-test.js

```diff
@@ -97,4 +97,19 @@ module('Integration | Modifier | autofocus', function(hooks) {
 
     assert.dom('[data-test-input-6]').isFocused('The custom selected element is focused');
   });
+
+  test('should focus a referenced element', async function (assert) {
+    await render(hbs`
+      <div {{autofocus this.input2}}>
+        <span>this is not a focusable element</span>
+        <button data-test-button>this is a button</button>
+        <input data-test-input-1 />
+        <input data-test-input-2 {{ref this "input2"}} />
```

Comment:
@tzellman why not put the `autofocus` modifier directly on this element instead? 🤔

Reply:
Ah, good question. Seriously. I believe at the time I had a use-case, but I think it is no longer valid.

I believe the use-case had to deal with a conditionally rendered element giving focus to another element (somewhere in the DOM but referenced via a property). An alternative way of handling is to call focus on the ref'd element on `did-insert` of the conditional element.

I'll close the PR. Thanks for reviewing!